stories/SimplePopup.js

```diff
@@ -17,14 +17,43 @@ const DefaultSimplePopup = () => {
       buttonClass: 'cancel-button',
       dataQeId: 'button-cancel',
     },
+  ];
+
+  return (
```

Comment:
please use a default props object and spread, along with overriding props, like we do in the tests. DRY. dont re-write a wrapper each time

Reply:
done